boards/BMS/BMS.ioc

```diff
@@ -1,8 +1,8 @@
 #MicroXplorer Configuration settings - do not modify
```

Comment:
Unfortunately the BMS board is stuck in the EDC so I can't check that these settings on hardware. But this is more or less the same as #614 so I have reasonable confidence in its correctness.